salt/metalk8s/kubeadm/init/addon/installed.sls

```diff
@@ -0,0 +1,11 @@
+Install pip package:
+  pkg.installed:
+    - name: python2-pip
+
+Install kubernetes python client:
+  pip.installed:
```

Comment:
No, don't install things using `pip`.

Reply:
`python2-kubernetes` exists in `extras`. If the Salt state can't work with it, then that needs to be fixed, not using some ancient version of upstream.